src/overlay/Peer.cpp

```diff
@@ -1746,35 +1728,13 @@ Peer::recvAuth(StellarMessage const& msg)
         return;
     }
 
-    bool enableBytes =
-        (mAppConnector.getConfig().OVERLAY_PROTOCOL_VERSION >=
-             Peer::FIRST_VERSION_SUPPORTING_FLOW_CONTROL_IN_BYTES &&
-         getRemoteOverlayVersion() >=
-             Peer::FIRST_VERSION_SUPPORTING_FLOW_CONTROL_IN_BYTES);
-    bool bothWantBytes =
```

Comment:
Should we still check if a peer has AUTH_MSG_FLAG_FLOW_CONTROL_BYTES_REQUESTED enabled, and drop it if not? This way we can add a drop reason to the error for clarity

Reply:
Good idea. Done in 9e862982a98e7f04fd2ce8b72618c7fe8977334b